src/modules/fstab/main.py

```diff
@@ -152,20 +152,22 @@ def generate_crypttab_line_info(self, partition):
         if not mapper_name or not luks_uuid:
             return None
 
-        if mount_point == "/":
+        if mount_point != "/":
```

Comment:
What I found is that the lines are never populated when the `mount_point == "/"` is checked. The commented part crypttab header text is inserted properly but no uncommented lines are ever written because the function returns 'None' before it can return the dictionary. 

In my case, mount_point is always "/". I am not sure what to do about this.